src/cli/create_command.zig

```diff
@@ -456,7 +456,7 @@ pub const CreateCommand = struct {
                 node.name = "Copying files";
                 progress.refresh();
 
-                const template_dir = std.fs.openDirAbsolute(filesystem.abs(&template_parts), .{}) catch |err| {
```

Comment:
I don't think `.iterate` does anything on macOS

I think this is unlikely to fix the issue

Reply:
> I don't think `.iterate` does anything on macOS
> 
> I think this is unlikely to fix the issue

You can repro this issue on linux and windows, but it does not fix the flakiness on macos

Reply:
the directory is iterated. iterate should be passed